agentMET4FOF/metrological_streams.py

```diff
@@ -60,12 +93,14 @@ class MetrologicalSineGenerator(MetrologicalDataStreamMET4FOF):
     to be supplied to the `set_generator_function`
 
     """
-    def __init__(self,sfreq = 500, F=5):
+
+    def __init__(self, sfreq=500, F=50):
         super().__init__()
-        self.set_metadata("SineGenerator","time","s",("Voltage"),("V"),"Simple sine wave generator")
+        self.set_metadata(device_id="SineGenerator", time_name="time", time_unit="s", quantity_names=("Voltage"),
+                          quantity_units=("V"), misc="Simple sine wave generator")
```

Comment:
Don't we want to provide the possibility to specify the metadata right at initialization via named parameters which default to the proposed values?

Reply:
We do that.